binding/springamqp/src/main/java/io/tracee/binding/springamqp/TraceeMessagePropertiesConverter.java

```diff
@@ -0,0 +1,74 @@
+package io.tracee.binding.springamqp;
+
+import com.rabbitmq.client.AMQP;
+import com.rabbitmq.client.Envelope;
+import io.tracee.Tracee;
+import io.tracee.TraceeBackend;
+import io.tracee.TraceeConstants;
+import io.tracee.configuration.TraceeFilterConfiguration;
+import io.tracee.transport.HttpHeaderTransport;
+import org.springframework.amqp.core.MessageProperties;
+import org.springframework.amqp.rabbit.support.DefaultMessagePropertiesConverter;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import static io.tracee.configuration.TraceeFilterConfiguration.Channel.IncomingResponse;
+import static io.tracee.configuration.TraceeFilterConfiguration.Channel.OutgoingRequest;
+
+public class TraceeMessagePropertiesConverter extends DefaultMessagePropertiesConverter {
+
+	private final TraceeBackend backend;
+	private final HttpHeaderTransport transportSerialization;
```

Comment:
Is there a reason for tracee context serialization? I read the Spring Javadoc and the AMQP-Header is able to deal with objects. Maybe we could store a Map directly like we do in the JMS-Implementation as well. (See `io.tracee.binding.jms.TraceeMessageListener`for example).